src/storybook/src/modul-components/components/overlay/overlay.stories.ts

```diff
@@ -15,6 +15,14 @@ export const defaultStory = () => ({
     template: `<div>
             <m-overlay title="Title" @save="save" @cancel="cancel">
                 <m-button slot="trigger">Open</m-button>
+                <m-dropdown class="m-u--margin-bottom--s" :filterable="true" :clear-model-on-selected-text="true" label="Vegetable" v-model="model1">
```

Comment:
Afin de facilité la maintenance et la cohérance globale des stories et test,  si on règle un bug dans le dropdown le test devrait être dans les stories du dropdown.

La stories pourrait avoir comme nom "drodown in overlay" dans dropdown.stories.ts

Reply:
Done!